### ISSUE TITLE
Feature Request: Sort Order

### ISSUE THREAD

**Issue by inanorbit** [IdreesInc/Waypoint#94]
I use Waypoint on folders which contain subfolders that are ordered by date. The default sort ends up with the newest folders at the bottom of the list, and the oldest at the top. Since I spend much more time on more recent files, it would be helpful to have at least the option to reverse the sort order so that newer folders are at the top. I have a lot of folders to sort through, so when I use Waypoint I end up doing a lot of scrolling.

I use another plugin called Custom File Explorer Sorting to achieve this in the file explorer, but this doesn't translate to the Waypoint output. Thanks!

**Comment by arranf** [comment]
I would also love this functionality!

**Comment by azazello** [comment]
Yes, would be great if waypoint respected Obsidian sort order set by the user
